src/mpid/ch4/shm/ipc/cma/cma_post.c

```diff
@@ -100,19 +100,23 @@ static int copy_iovs(pid_t src_pid, MPI_Aint src_data_sz,
 
     MPI_Aint i_src = 0, i_dst = 0;
     MPI_Aint cur_offset = 0;
+    bool got_err_truncate = false;
 
+#define MAX_IOV_SEG_SIZE 0x7ffff000
```

Comment:
is there some external source of this limit we can link to in a comment or something?

Reply:
I didn't find any. I'll add a FIXME comment.